### PR TITLE
Update streams

### PR DESCRIPTION
**Pull request by Alstruit**
Some cleanups.

### CODE REVIEW

File: streams/us_local.m3u [hunk 1]
```diff
@@ -13,8 +13,6 @@ https://stream.krgv.com/krgv-english/krgv-english.smil/playlist.m3u8
 https://content.uplynk.com/channel/64ca339f04964a5e961c3207a7771bf1.m3u8
 #EXTINF:-1 tvg-id="KABCDT1.us",ABC 7 Los Angeles CA (KABC-TV) (720p)
 https://content.uplynk.com/channel/ext/2118d9222a87420ab69223af9cfa0a0f/kabc_24x7_news.m3u8
-#EXTINF:-1 tvg-id="WJLADT1.us",ABC 7 Washington DC (WJLA) (720p)
-https://content.uplynk.com/40cec2bf074c40f08932da03ab4510be.m3u8
```
Comment (BellezaEmporium) [inline streams/us_local.m3u:17]: https://content.uplynk.com/d967631805a947b79e4c640bf56c90af.m3u8

File: streams/us_local.m3u [hunk 2]
```diff
@@ -100,44 +98,30 @@ https://reflect-stream15-capsmedia.cablecast.tv/live/live.m3u8
 #EXTINF:-1 tvg-id="CityTVCarlsbad.us",Carlsbad City TV Channel (Carlsbad CA) (720p)
 https://edge-f.swagit.com/live/carlsbadca/live-3-a-1/playlist.m3u8
 #EXTINF:-1 tvg-id="KMTVDT1.us",CBS 3 Omaha NE (KMTV-TV) (720p)
-https://content.uplynk.com/4a09fbea28ef4f32bce095e9eae04bd8.m3u8
-#EXTINF:-1 tvg-id="KMTVDT1.us",CBS 3 Omaha NE (KMTV-TV) (720p)
 https://content.uplynk.com/channel/328d1434fb51476cb6567c74d5b2cc70.m3u8
-#EXTINF:-1 tvg-id="WRGBDT1.us",CBS 6 Albany NY (WRGB) (720p)
-https://content.uplynk.com/channel/bba3e7da884a49bba96341ecf5128f0f.m3u8
 #EXTINF:-1 tvg-id="WUSADT1.us",CBS 9 Washington DC (WUSA) (1080p)
 https://video.tegnaone.com/live/wusa/v1/master/f9c1bf9ffd6ac86b6173a7c169ff6e3f4efbd693/WUSA-2/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us_local.m3u:103]: Alternate : https://livevideo.tegnadigital.com/khou/v1/master/f9c1bf9ffd6ac86b6173a7c169ff6e3f4efbd693/WUSA-2/index.m3u8

File: streams/us_local.m3u [hunk 3]
```diff
@@ -248,12 +232,8 @@ https://2-fss-1.streamhoster.com/pl_122/200562-1176456-1/playlist.m3u8
 https://reflect-golden-co.cablecast.tv/live-3/live/live.m3u8
 #EXTINF:-1 tvg-id="GreensboroTV.us",Greensboro TV (Greensboro NC) (720p)
 https://lin12.isilive.ca/live/greensboro/GTN/index.m3u8
-#EXTINF:-1 tvg-id="WTXLDT3.us",Grit Tallahassee (480p) [Not 24/7]
-https://content.uplynk.com/channel/c66d1d2d9e2c46cea1cf22a33f6ca488.m3u8
```
Comment (BellezaEmporium) [inline streams/us_local.m3u:252]: works

Comment (Alstruit) [reply streams/us_local.m3u:252]: That is for [WTXL-DT1, ABC Tallahassee](https://github.com/iptv-org/iptv/pull/15484/files/85f3741832c4b7cd630e45fb1375541d82c9aa2f#diff-6bd195e23caaa57313e2bc2ee0c0c12bc5ae2d0217e1ea1514b70a5dfe8b55ffR33).

File: streams/us_local.m3u (hunk 1 above)
Comment (Alstruit) [reply streams/us_local.m3u:17]: Tokenized.

**Review by Dum4G — DISMISSED** [review]

**Review by BellezaEmporium — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**